scripts/multi_ldap_change.php

```diff
@@ -0,0 +1,135 @@
+<?php
+#==============================================================================
+# Includes
+#==============================================================================
+require_once(__DIR__."/../conf/config.inc.php");
+require_once(__DIR__."/../lib/functions.inc.php");
+
+#==============================================================================
+# Action
+#==============================================================================
+
+$result = "";
+$return = Array();
+$error_code = 1;
+$login = $argv[1];
+$newpassword = $argv[2];
+$oldpassword = '';
+
+$log_file = fopen(__DIR__.'/multi_ldap_change.log', 'a+');
+fwrite($log_file, "Change '$login' password...\n");
+
+foreach ($secondaries_ldap as $s_ldap) {
+    # Connect to LDAP
+    $ldap = ldap_connect($s_ldap['ldap_url']);
+    ldap_set_option($ldap, LDAP_OPT_PROTOCOL_VERSION, 3);
+    ldap_set_option($ldap, LDAP_OPT_REFERRALS, 0);
+    if ( $ldap_starttls && !ldap_start_tls($ldap) ) {
+        $result = "ldaperror";
+        fwrite($log_file, "LDAP - Unable to use StartTLS");
+    } else {
+
+    # Bind
+    if ( isset($ldap_binddn) && isset($ldap_bindpw) ) {
+        $bind = ldap_bind($ldap, $ldap_binddn, $ldap_bindpw);
+    } else {
+        $bind = ldap_bind($ldap);
+    }
+
```

Comment:
You suppose here that connection settings to secondary repositories are the same that the main LDAP server.

Maybe your config object  `$s_ldap` should contain all connection parameters.